tests/codeception/acceptance/Media/MediaFormsCest.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/**
+ * Tests for various media form functionality.
+ */
+class MediaFormsCest {
+
+  /**
+   * Test embeddables form alters
+   */
+  public function testFormAlters(AcceptanceTester $I) {
+    $I->logInWithRole('administrator');
+    $I->amOnPage('/media/add/embeddable');
+    $I->seeLink('request support', 'https://stanford.service-now.com/it_services?id=sc_cat_item&sys_id=83daed294f4143009a9a97411310c70a');
```

Comment:
This link won't be visible for administrators. You should use composer to target the `stanford_profile_helper` branch related to these tests to verify they pass.

Reply:
fixed.